### ISSUE TITLE
Return error when barcode could not be generated

### ISSUE THREAD

**Issue by thoean** [metafloor/bwip-js#60]
When the text does not fit in the provided size of the QR Code, then an empty image is returned. I'd expect an error message, which would be easier to handle.

The following unit test succeeds:

```javascript
      it('width / height test', function (done) {

        let options = {
            bcid: 'qrcode',
            text: 'very-long-text-that-will-not-fit-in-10px-by-10px',
            height: 10,
            width: 10,
            scale: 1
        };

        bwipjs.toBuffer(options, (err, result) => {
            let width = result.readUInt32BE(16);
            let height = result.readUInt32BE(20);
            expect(width).toBe(0);
            expect(height).toBe(0);
            done();
        });
    });
```

**Comment by metafloor** [comment]
The bug is due to the modules of the symbol being scaled to zero width and height.  I have pushed a commit to the project's github master branch (file bwipjs.js) which throws when a module is zero sized.  I will push to npm later today when I have time to update the online API as well.

**Comment by thoean** [comment]
I confirm that this is fixed with version `1.2.3`. Thanks for the quick turnaround.

To specifically test for this case, one could inspect the error message like this:

```javascript
        this.bwipjs.toBuffer(options, (err, buffer) => {
            if (err) {
                if (err.search(/Image scaled to zero size/i) > -1) {
                    reject("image error" + err);
                }
                else {
                    reject("Other error" + err);
                }
            } else {
                resolve(buffer);
            }
        });
```
